include/mgba/internal/gba/io.h

```diff
@@ -157,6 +157,31 @@ enum GBAIORegisters {
 	REG_DEBUG_STRING = 0xFFF600,
 	REG_DEBUG_FLAGS = 0xFFF700,
 	REG_DEBUG_ENABLE = 0xFFF780,
+
+	// Extensions
+	REG_HWEX_ENABLE = 0x400A00,
```

Comment:
This base seems arbitrary. How did you choose it?

Reply:
I didn't know what to put, so I chose something in the middle. Didn't want to put it at the end so its easy to add more and also didn't want to put it close to the real registers. Any suggestions?